doc/api/util.md

```diff
@@ -251,7 +252,16 @@ util.format('%s:%s', 'foo');
 ```
 
 Values that are not part of the format string are formatted using
```

Comment:
suggest: `s/part of/consumed by/` -- since no value is part of the format string, except the format string itself.

Reply:
I went through the descriptions again and updated a bit more. PTAL.